instrumentation/spring/spring-webmvc-3.1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/springwebmvc/SpringWebMvcTracer.java

```diff
@@ -91,10 +97,12 @@ private String spanNameOnRender(ModelAndView mv) {
   }
 
   private void onRender(Span span, ModelAndView mv) {
-    span.setAttribute("spring-webmvc.view.name", mv.getViewName());
-    View view = mv.getView();
-    if (view != null) {
-      span.setAttribute("spring-webmvc.view.type", spanNameForClass(view.getClass()));
+    if (captureExperimentalSpanAttributes) {
+      span.setAttribute("spring-webmvc.view.name", mv.getViewName());
+      View view = mv.getView();
+      if (view != null) {
+        span.setAttribute("spring-webmvc.view.type", spanNameForClass(view.getClass()));
```

Comment:
Why the hell anybody could want this??

Reply:
step 1: move to experimental
step 2: eliminate(?)